page-stories.hbs

```diff
@@ -2,6 +2,20 @@
 
 <div class="gh-page">
     <div class="gh-container">
+        <div class="filter">
+            <div class="dropdown">
+                <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    Filter
+                </button>
+                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
```

Comment:
Could you add two dropdown menus, one with a filter for just the *external* tags, and another for just Conversations, Insights & Reflections? (Feel free to hardcode the latter, as it's assumed throughout the entire codebase that these three internal tags exist.)